vod/manifest_utils.h

```diff
@@ -46,12 +46,12 @@ typedef struct {
 // functions
 vod_status_t manifest_utils_build_request_params_string(
 	request_context_t* request_context,
-	uint32_t* has_tracks,
+	track_mask_t has_tracks[MEDIA_TYPE_COUNT],
```

Comment:
i prefer track_mask_t* instead of []'s (also, that's how it's defined in the C file)